apps/sms/views/conversation/js/conversation.js

```diff
@@ -2009,7 +2077,8 @@ var ConversationView = {
     var bubble;
 
     do {
-      if (node.classList.contains('bubble')) {
+      // A Text node does not have a classList property
+      if (node.classList && node.classList.contains('bubble')) {
```

Comment:
I really don't reproduce :/ I tried both on device and in Firefox... Can you check once again ?
Maybe you had the issue on unit tests or integration tests ?

And if you still have the issue, maybe do an early continue like `if (!node.classList) { continue; }` at the start of the loop (with appropriate indentation) ? What do you think ?